interfaces/prometheus_scrape/v0/README.md

```diff
@@ -52,7 +52,7 @@ Both the Requirer and the provider need to adhere to a certain set of criterias
 
 ```yaml
 application-data:
-  alert_rules: {
+  alert_rules: '{
```

Comment:
this would be easier to express if we refactored the whole interface to use the modern pydantic `schema.py`.
This looks like a purely cosmetic change, so I'm fine with it, but at some point we should probably do that instead.

Reply:
Ack, understood (will close out this comment since it currently claims to be blocking, although your comment clarifies that it isn't).